transcoder.py

```diff
@@ -474,9 +474,8 @@ def copyCoverFiles(lossyTree):
             copyFile = True
 
         if copyFile:
-          log('- copying ' + sourceCoverFullFileName + ' to ' + lossyCoverFullFileName) 
+          log('- copy cover to ' + lossyTree) 
```

Comment:
By ommitting value of lossyCoverFullFileName, it is not clear which file is processed